portfolio/src/main/webapp/index.html

```diff
@@ -9,6 +9,9 @@
   <body>
     <div id="content">
       <h1>Caleb Baines Portfolio</h1>
+      <img src="./images/caleb_pic.jpg" alt="Picture of Caleb"
+      width="200" height="225"
+      />
```

Comment:
Feel free to condense these 3 lines down to one, it looks like there's enough space

Reply:
+1

Reply:
```suggestion
     <img src="./images/caleb_pic.jpg" alt="Picture of Caleb" width="200" height="225" />
```